app/assets/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -330,6 +338,16 @@ class DatasetImportView extends React.PureComponent<Props, State> {
               })(<Input.TextArea rows={20} style={jsonEditStyle} />)}
             </FormItem>
             {this.props.isEditingMode ? this.getEditModeComponents() : null}
+            <FormItem label="Allowed Teams">
+              <TeamAssignment
```

Comment:
This is not exactly how Antd Forms work. It is best practive to wrap every from item in a wrapper `getFieldDecorator`. This allows the form to easily serialize all fields into a JS object on submit, validate the fields, and "prefill" the from if needed. The wrapper will pass a `value` prop to the wrapped component so it can access the value. In our case, the whole dataset object is passed to the form on load and all form fields are pre filled when editting. See line 132.

See also https://ant.design/components/form/#this.props.form.getFieldDecorator(id,-options).

Reply:
Maybe you can explain this to me on monday in greater detail